planb/update_cluster.py

```diff
@@ -533,6 +527,9 @@ def update_cluster(options: dict):
         # TODO: user should hit Ctrl-c to cancel everything
         # don't ask again if resuming after crash
         if len(instances) > 1:
+            logger.warn("-----------------------------------------------")
+            logger.warn("!!! Check your monitoring before proceeding !!!")
```

Comment:
Should we be more specific? Like "Do all nodes agree on no node is down?"

Reply:
Well, I think it is more general: e.g. if the latencies were hurt or something else the user cares about.